### ISSUE TITLE
add `isTemplate` a `usedTemplate` to the Search box

### ISSUE THREAD

**Issue by adammertel** [DISSINET/InkVisitor#750]

**Comment by Ptrhnk** [comment]
@davidzbiral What about this, is it needed to version 1.3.0?

**Comment by davidzbiral** [comment]
@Ptrhnk: Yes, it is needed in 1.3.0. (I can explain why if needed - because some ex-Concepts will be imported as Statement templates.)

**Comment by adammertel** [comment]
`isTemplate` is probably not needed as we have the Template box now. But probably storing the information about the template used and having this search option in the Search box is sufficient.

**Comment by adammertel** [comment]
#956  is a precondition here

**Comment by adammertel** [comment]
and also #960

**Comment by davidzbiral** [comment]
@adammertel: "isTemplate is probably not needed as we have the Template box now": ok, but does one recognize in the JSON structure that something is a template?

**Comment by adammertel** [comment]
The entity tag will have different graphics.
